rfc/attributes.md

```diff
@@ -35,11 +35,11 @@ ATTRIBUTE       ::= 'param' MODIFIER? | 'reader' MODIFIER? | 'writer' MODIFIER?
                  |  'predicate' MODIFIER?  | 'name' MODIFIER? | HANDLES
 ATTRIBUTES      ::= { ATTRIBUTE }
 HANDLES         ::= 'handles' '('
-                                    IDENTIFIER { ',' IDENTIFIER }    # list of methods this slot handles
-                                 |  PAIR { ',' PAIR }                # map of methods (to, from) this slot handles
-                                 | '*'                               # this slot handles all unknown methods, but inheritance takes precedence
+                                    DELEGATION { ',' DELEGATION }    # this slot handles methods by delegating to its value (an object)
```

Comment:
Do we need the comma, or is whitespace sufficient? If we need the comma, for a simple list it might still look like Perl. And the proposed syntax doesn't allow a trailing comma.